config/splat.pspeu.stnz1.yaml

```diff
@@ -0,0 +1,111 @@
+options:
+  platform: psp
+  basename: stnz1
+  base_path: ..
+  build_path: build/pspeu
+  target_path: disks/pspeu/PSP_GAME/USRDIR/res/ps/PSPBIN/nz1.bin
+  asm_path: asm/pspeu/st/nz1_psp
+  asset_path: assets/st/nz1
+  src_path: src/st/nz1
```

Comment:
Can we please keep this pathing consistent with the other pxp overlays?

Reply:
I didn't realize this differed from what `make-config.py` produced.